exercises/beer-song/inputdata.txt

```diff
@@ -0,0 +1,7 @@
+99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n
+44 bottles of beer on the wall, 44 bottles of beer.\nTake one down and pass it around, 43 bottles of beer on the wall.\n
```

Comment:
This will make the test code more complicated to read in the input txt.  The reading code will have to loop essentially over a set number of lines per each test.  I am hoping to keep the file as useful information for the test runner, not necessarily something that the student could/might use as input to their code.

As it is now a single read line is essentially occuring for each test.